StellarObject.cpp

```diff
@@ -234,8 +234,8 @@ double StellarObject::Period() const
 // Get the radius of this planet, i.e. how close you must be to land.
 double StellarObject::Radius() const
 {
-    auto it = INFO.find(sprite);
-    return (it != INFO.end() ? it->second.radius : 40);
+    const auto &it = INFO.find(sprite);
```

Comment:
And, same thing here, again. My concern is just that when I see a const reference in code, I usually assume it's a reference to a (semi)-permanent object, not just to a temporary return value of a function.